agent/command/assume_ec2_role_test.go

```diff
@@ -50,6 +50,8 @@ func TestEc2AssumeRoleExecute(t *testing.T) {
 	c := &ec2AssumeRole{}
 	for testName, testCase := range map[string]func(ctx context.Context, t *testing.T, c *ec2AssumeRole, comm *client.Mock, logger client.LoggerProducer, conf *internal.TaskConfig){
 		"FailsWithNoARN": func(ctx context.Context, t *testing.T, c *ec2AssumeRole, comm *client.Mock, logger client.LoggerProducer, conf *internal.TaskConfig) {
+			c.RoleARN = ""
```

Comment:
If we just move defining c into t.Run, then we guarantee that it's always new. (We also can move anything that isn't in that section out of the for loop func() definition, since anything defined outside of the for loop is accessible without needing to pass it in)

Reply:
done